### DISCUSSION TITLE
Ajuste no Grid Layout - 1024px

### DISCUSSION THREAD

**Discussion by msagrilo** [Telefonica/mistica-design#1883] [Proposals]
### Goal

O objetivo é fazer um ajuste no grid layout do Mística, na documentação temos que o grid é fluído, porém quando aplicado em um frame do Figma, ele está fixo no breakpoint de 1024-1367.

![image](https://github.com/user-attachments/assets/e2194c70-3516-49b5-a0e8-4b0bed796381)

### Context

Quando aplicamos o grid 1024px do Mística ele não se comporta como na documentação.

Como está hoje:

![image](https://github.com/user-attachments/assets/aa2d8dfb-71ce-4a74-89d2-030854bdbf51)

Como deveria ficar:

![image](https://github.com/user-attachments/assets/8675be84-ffd2-49d3-b05c-34afcd50bc13)

As colunas estão com tamanho fixo, mas deveriam ser fluid até 1367px com 40px de margin.

### Impact

Impactará na consisitência da documentação e o grid passará a agir corretamente.

### Discussion Questions

Ainda sobre o Grid Layout, temos planos para uma evolução dele com as variables?

### Do you have a visual proposal?

[Ajuste no Grid Layout - 1024px - POC](https://www.figma.com/design/HxKasHOfRW8iV5lgZSiwpw/Estudos-Sagrilo?node-id=534-1018&t=oK12fJCMxiobOCaO-1)

**Comment by yceballost** [comment]
issue created https://github.com/Telefonica/mistica-design/issues/1886 

Recuerda incluir el link a vuestro Figma :)

Obrigado!

**Reply by msagrilo** [reply]
Atualizei a discussion com o link do Figma.

Obrigado!
